53638: Add text to Storage/Create Pool section about Dashboard summary cards for new pools. (#656)
freenas/freenas-docs: userguide/storage.rst

:guilabel:`DONE`.

> The screen refreshes after the pool is created, showing the new pool

Reviewer: Didn't we remove this sentence entirely in one version?  It does not really add any information helpful to the user.

Author: I checked all open PRs and merged angulargui back into this branch. There are no changes to this sentence.

:guilabel:`DONE`.

The screen refreshes after the pool is created, showing the new pool
in
:menuselection:`Storage --> Pools`.

> In the example shown in :numref:`Figure %s <zfs_vol_fig>`, the created

Reviewer: ```suggestion
:numref:`Figure %s <zfs_vol_fig>` shows the new *pool1*.
```

> pool is named *pool1*. The description indicates that it is healthy and

Reviewer: Delete this entire sentence, from "The description" to "used and free."
```suggestion

```

   Viewing Pools

> Creating a pool also adds a card to the

Reviewer: I thought Dashboard things were called "widgets".

```suggestion
Creating a pool adds a card to the
```

:menuselection:`Dashboard`.
Available space, disk details, and pool status is shown on the card.

> The background color of the card also reflects the pool status:

Reviewer: ```suggestion
The background color of the card indicates the pool status:
```

> * Green: healthy or locked

Author: Nope, the card only displays a single status as text: healthy or locked.
